juju/model.py

```diff
@@ -31,8 +31,21 @@ def relation(self, relation_name):
             # ideally integrating the error catching with Juju's mechanisms.
             raise TooManyRelatedApps(relation_name, num_related, 1)
 
+    def get_relation_by_id(self, relation_name, relation_id):
```

Comment:
The prior review had a note about this method name which was agreed and resolved, but it's still here.